### PR TITLE
4320 e2e scenario 2.2 child parent service

### PR DESCRIPTION
**Pull request by LauraStordeur**
# Description

E2E scenario for the child-parent service instances.

closes *#4320*

# Self Check:

Strike through any lines that are not applicable (`~~line~~`) then check the box

- [ ] Attached issue to pull request
- [X] Changelog entry
- [X] Code is clear and sufficiently documented
- [X] Sufficient test cases (reproduces the bug/tests the requested feature)

### CODE REVIEW

File: cypress/e2e/scenario-2.2-child-parent-service.cy.js [hunk 1]
```diff
@@ -0,0 +1,201 @@
...
+
+    // open row from element
+    cy.get("#expand-toggle0", { timeout: 20000 }).click();
+    // try delete item (Should not be possible)
+    cy.get(".pf-c-description-list").contains("Delete").click();
+    cy.get(".pf-c-modal-box__title-text").should("contain", "Delete instance");
+    cy.get(".pf-c-form__actions").contains("Yes").click();
+
+    // check status change before compile
+    cy.wait("@DeleteParentService");
```
Comment (matborowczyk) [inline cypress/e2e/scenario-2.2-child-parent-service.cy.js:167]: aren't we getting rid of these awaits?

File: cypress/e2e/scenario-2.1-basic-service.cy.js [hunk 2]
```diff
@@ -234,23 +229,21 @@ describe("Scenario 2.1 Service Catalog - basic-service", () => {
...
-    // wait to check if compile is done
-    waitForCompile();
-    cy.wait("@GetServiceInventory");
-
     // expand first row
-    cy.get("#expand-toggle0").click();
+    cy.get("#expand-toggle0", { timeout: 20000 }).click();
 
-    // There might be still a compile running because of the edit action. So we do a check if one is pending.
     waitForCompile();
```
Comment (matborowczyk) [inline cypress/e2e/scenario-2.1-basic-service.cy.js:235]: why this one stays, and other is deleted?

File: cypress/e2e/scenario-2.2-child-parent-service.cy.js [hunk 3]
```diff
@@ -0,0 +1,201 @@
...
+    // try delete item (Should be possible)
+    cy.get(".pf-c-description-list", { timeout: 20000 })
+      .contains("Delete")
+      .click();
+    cy.get(".pf-c-modal-box__title-text").should("contain", "Delete instance");
+    cy.get(".pf-c-form__actions").contains("Yes").click();
+
+    cy.wait("@DeleteChildService").its("response.statusCode").should("eq", 200);
+
+    cy.wait("@GetParentInventory");
```
Comment (matborowczyk) [inline cypress/e2e/scenario-2.2-child-parent-service.cy.js:197]: as above

File: cypress/e2e/scenario-2.1-basic-service.cy.js (hunk 2 above)
Comment (LauraStordeur) [reply cypress/e2e/scenario-2.1-basic-service.cy.js:235]: cause I'm sure 100% that one would be running at least. So it's a bit of a check to see if a compile is triggered if that makes sense.

File: cypress/e2e/scenario-2.2-child-parent-service.cy.js (hunk 1 above)
Comment (LauraStordeur) [reply cypress/e2e/scenario-2.2-child-parent-service.cy.js:167]: I'll test if removing them also works !

File: cypress/e2e/scenario-2.1-basic-service.cy.js (hunk 2 above)
Comment (matborowczyk) [reply cypress/e2e/scenario-2.1-basic-service.cy.js:235]: Makes sense :D

**Review by matborowczyk — APPROVED** [review]

**Pull request closed without merging**